docs-website-src/content/blockers.md

```diff
@@ -47,29 +52,40 @@ You can discover many of these issues by downloading `elevate-cpanel` and runnin
+* **Name Server**
+  * cPanel provides support for a myriad of name servers. (MyDNS, NSD, BIND, PowerDNS). On RHEL 8 based distributions, it is preferred that you always be on PowerDNS.
   * Mitigation: `/scripts/setupnameserver powerdns`
 * **MySQL**
-  * We will upgrade you automatically to MariaDB 10.6 during elevation if you do not do this in advance.
+  * If the version of MySQL/MariaDB installed on the system is not supported on RHEL 8 based distributions, you **must** upgrade to a supported version. If cPanel manages the MySQL installation, we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation.
+  * Elevation will block if a MySQL upgrade is in progress.
+  * The system **must** not be setup to use a remote database server.
 * Some **EA4 packages** are not supported on AlmaLinux 8.
   * Example: PHP versions 5.4 through 7.1 are available on CentOS 7 but not AlmaLinux 8. You would need to remove these packages before upgrading. Doing so might impact your system users. Proceed with caution.
 * The system **must** be able to control the boot process by changing the GRUB2 configuration.
-  * The reason for this is that the framework which performs the upgrade of distro-provided software needs to be able to run a custom early boot environment (initrd) in order to safely upgrade the distro.
+  * The reason for this is that the Leapp framework, which performs the upgrade of distribution-provided software, needs to be able to run a custom early boot environment (initrd) in order to safely upgrade the distribution.
   * We check for this by seeing whether the kernel the system is currently running is the same version as that which the system believes is the default boot option.
-* Your machine has multiple network interface cards (NICs) using kernel-names (`ethX`).
+  * We also check that there is a valid GRUB2 config.
+* We block if your machine has multiple network interface cards (NICs) using kernel-names (`ethX`).
   * Since `ethX` style names are automatically assigned by the kernel, there is no guarantee that this name will remain the same upon upgrade to a new kernel version tier.
-  * The "default" approach in `network-scripts` config files of specificying NICs by `DEVICE` can cause issues due to the above.
+  * The "default" approach in `network-scripts` config files of specifying NICs by `DEVICE` can cause issues due to the above.
   * A more in-depth explanation of *why* this is a problem (and what to do about it) can be found at [freedesktop.org](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/).
-  * One way to prevent these isssues is to assign a name you want in the configuration and re-initialize NICs ahead of time.
+  * One way to prevent these issues is to assign a name you want in the configuration and re-initialize NICs ahead of time.
+* Running the system in a container-like environment is not supported.
```

Comment:
not docker. Docker is not supported. More like Virtuozzo. I'm not actually certain if there are other examples we support.

Reply:
Since Virtuozzo is covered in more detail in the --no-leapp/almaconvert8 documentation, and, if there are no other valid examples, should 'Running the system in a container-like environment is not supported.' be included in this section?

Reply:
or maybe provide Virtuozzo as an example with a link to their solution: https://docs.virtuozzo.com/virtuozzo_hybrid_server_7_users_guide/advanced-tasks/converting-containers-with-almaconvert8.html

Reply:
These are the ones we look for and block on:
      virtuozzo
      vzcontainer
      lxc
      virtualiron
      vserver

Reply:
I'm fine with the docs without an example. It wouldn't hurt (in a different PR) for us to clearly document the vz8convert script and advertise it for people looking for it. Feel free to open a case @annes449

Reply:
Based on our discussion in slack, I'll leave the comment as-is.